src/Hydration.js

```diff
@@ -19,6 +20,15 @@ class Hydration extends UserFitness {
   returnWeeklyNumOunces() {
     return this.userData.slice(-7).map(day => day.numOunces);
   }
+  
+  returnFriendsDailyFluidOunces() {
```

Comment:
this function looks identical to the function on line 9 - returnAverageFluidOunces() what is the difference between these two?

Reply:
I was able to remove returnFriendsDailyFluidOunces() and reuse the same function. Good eye